lib/chef/application.rb

```diff
@@ -102,6 +102,12 @@ def load_config_file
       Chef::Config.merge!(config)
     end
 
+    def set_specific_recipes
+      Chef::Config[:specific_recipes] =
+        cli_arguments.map { |file| File.expand_path(file) } if
+        cli_arguments.respond_to?(:empty) && !cli_arguments.empty?
```

Comment:
Were you having issues where `cli_arguments` wasn't responding to empty?  That method should be provided by mixlib-cli, so I would think we can trust it.